doc/developers/contributing.rst

```diff
@@ -952,6 +952,22 @@ The easiest and recommended way to accomplish this is to
 All logic behind estimator parameters,
 like translating string arguments into functions, should be done in ``fit``.
 
+Parameters can be passed to the ``fit`` method. Generally, these parameters
+should be restricted to variables that need to be sliced during
+cross-validation. These variables would be either arrays with shape=[N] or
```

Comment:
Ok! The change has been made toe clarify N=n_samples. Please review when you have a chance.